test/parallel/test-net-server-keepalive.js

```diff
@@ -7,6 +7,17 @@ const server = net.createServer({
   keepAlive: true,
   keepAliveInitialDelay: 1000
 }, common.mustCall((socket) => {
+  const symbols = Object.getOwnPropertySymbols(socket);
```

Comment:
Instead of checking internal values, can't we just test this by making a local TCP connection? That would be more foolproof and less dependent upon how settings are stored/used internally.

Reply:
> Instead of checking internal values, can't we just test this by making a local TCP connection? That would be more foolproof and less dependent upon how settings are stored/used internally.

Thanks for review. I have updated the code, please help review again.